breathe/renderer/sphinxrenderer.py

```diff
@@ -1024,6 +1024,12 @@ def render_derivedcompoundref(node):
         addnode('innerclass', lambda: self.render_iterable(node.innerclass))
         addnode('innernamespace', lambda: self.render_iterable(node.innernamespace))
 
+        if 'inner' in options:
+            for node in node.innergroup:
```

Comment:
In the event that `inner` is requested but there are no inner groups, should we emit a warning along the lines of `inner groups requested for group {group name} but no inner groups found!`?

Reply:
Sure, and there's also the other way around. If there are `innergroup`s but no `inner` keyword was given in the options. To preserve compatibility this should obviously not trigger a warning. I am honestly not sure if someone would like the behavior of generating a warning in the case you pointed, but I will assume so and update accordingly. It can always be removed later! Btw, adding an `inner` in the documentation set I am working on, already triggers lots and lots of warnings because of the double references, because we basically had to create a reference to each group individually due to the lack of innergroup parsing.

Reply:
Thinking a bit better about this, turns out that `options` is part of a  "global" context in a sense. So if I have an innergroup B inside another group A, and I set `inner`, both A and B will be checked for the existence of innergroups. This will end up being an empty list for the innergroup B (assuming it has no innergroups itself!); but it should not trigger a warning.